content/topics/kotlin/classes/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Classes 
+ready: true
+prerequisites:
+  hard: ["topics/kotlin/basic-control-flow"]
+  soft: []
+
+---
+## Classes
+Classes in Kotlin are declared using the keyword class:
+
+````
+class Invoice {
+    }
+````
+
+Constructors
+A class in Kotlin can have a primary constructor and one or more secondary constructors.
+
+
+````
+class Person constructor(firstName: String) {
+    }
+
+````
+
+Secondary Constructors
```

Comment:
Also this. I think the html would look like:
```
<p> Secondary Constructors The class can also declare secondary constructors, which are prefixed with constructor:
</p>
```

There are a few other places where this is the case.

Better would be:

```
### Secondary Constructors
etc
```

Reply:
Yea I missed a few was typing really fast.